src/native/fs/fs_napi.cpp

```diff
@@ -1387,17 +1429,17 @@ void
 fs_napi(Napi::Env env, Napi::Object exports)
 {
     auto exports_fs = Napi::Object::New(env);
+    uv_lib_t *lib = (uv_lib_t*) malloc(sizeof(uv_lib_t));
```

Comment:
@eshelmarc Why did you move this malloc outside of the env condition of GPFS_DL_PATH?
Is there a problem to keep it where it was?

Reply:
no, moved it back